src/app/systelab-components/chip-button/chip-button.component.ts

```diff
@@ -15,6 +15,8 @@ export class ChipButtonComponent {
 
 	@Output() public changeButton = new EventEmitter();
 	@Output() public selectButton = new EventEmitter();
+	@Output() public buttonRemoved = new EventEmitter();
```

Comment:
Document the EventEmitters in the .md file

Reply:
added